src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/AvslåVedtak/AvslåVedtakForm.tsx

```diff
@@ -49,30 +52,32 @@ const AvslåVedtakForm: React.FC<Props> = ({
 
     return (
         <>
-            <StyledForm onSubmit={lagreVedtak}>
-                {skalVelgeÅrsak && (
-                    <SelectAvslagÅrsak
-                        avslagÅrsak={avslagÅrsak}
-                        settAvslagÅrsak={settAvslagÅrsak}
-                        feilmelding={feilmeldingÅrsak}
+            <form onSubmit={lagreVedtak}>
+                <Container>
```

Comment:
For å slippe å sette margin-top i alle children kan `Container` gjøres om til en flex-box med gap=xrem. Da styres all spacing fra utsiden, vs. at man må endre på hver enkelt child.

Reply:
Dette gjelder alle forms og burde kanskje også være en container med liknende egenskaper på hele siden